srcpkgs/bitcoin/template

```diff
@@ -50,6 +50,7 @@ bitcoin-devel_package() {
 	}
 }
 
+if [ "$build_option_gui" ]; then
```

Comment:
please just use subpackages for this as it was before.

Reply:
If I left it as it was, in some cases it would say files not found because if being cross-built, they don't get built.

Reply:
I just meant instead of putting the function in an if block, add the sub-package to the subpackages variable conditionally.